htslib/hts.h

```diff
@@ -366,6 +366,7 @@ extern "C" {
 
     int hts_idx_get_stat(const hts_idx_t* idx, int tid, uint64_t* mapped, uint64_t* unmapped);
     uint64_t hts_idx_get_n_no_coor(const hts_idx_t* idx);
+    char *hts_idx_getfn(const char *fn, const char *ext);
```

Comment:
This is an htslib-internal function -- it has weird side-effects due to calling `test_and_fetch()`.  So I think we need to come up with a better solution than declaring it in a public header...